distributions/otelcol-ebpf-profiler/.goreleaser.yaml

```diff
@@ -0,0 +1,96 @@
+version: 2
+project_name: opentelemetry-collector-releases
+env:
+  - COSIGN_YES=true
+  - LD_FLAGS=-s -w
```

Comment:
The eBPF Profiler uses some more LD flags - would this be the correct place to add them as well?
https://github.com/open-telemetry/opentelemetry-ebpf-profiler/blob/ead430a15d497736841ee58436ed4f9a6dc86703/Makefile#L43-L46

Reply:
Done.
Not that the vc flags will have similar data added by goreleaser.
https://goreleaser.com/cookbooks/using-main.version/